tools/kdc/a

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can you change the filename to something more meaningful? Or was this added by mistake?

Reply:
Oh yeah, removed 😞